var/spack/repos/builtin/packages/libspatialite/package.py

```diff
@@ -18,33 +16,22 @@ class Libspatialite(AutotoolsPackage):
 
     license("MPL-1.1")
 
+    version("5.1.0", sha256="43be2dd349daffe016dd1400c5d11285828c22fea35ca5109f21f3ed50605080")
     version("5.0.1", sha256="eecbc94311c78012d059ebc0fae86ea5ef6eecb13303e6e82b3753c1b3409e98")
     version("5.0.0", sha256="7b7fd70243f5a0b175696d87c46dde0ace030eacc27f39241c24bac5dfac6dac")
-    # Must download manually from:
-    #     https://www.gaia-gis.it/fossil/libspatialite/info/c7f67038bf06d98d
-    # For instructions on the file:// below..
-    #     https://github.com/spack/spack/issues/2489
-    version(
-        "5.0.0.2.c7f67038bf",
-        sha256="f8100f71b769c7db066c6f938af6b00e920e4b90ac14c00a4f3ed7171565caab",
-        url="file://%s/SpatiaLite-c7f67038bf.tar.gz" % os.getcwd(),
-    )
-
-    version(
-        "5.0.0-beta0", sha256="caacf5378a5cfab9b8e98bb361e2b592e714e21f5c152b795df80d0ab1da1c42"
-    )
```

Comment:
Versions should be deprecated before removing them: https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions